DESCRIPTION

```diff
@@ -4,7 +4,7 @@ Title: Extension of `data.frame`
 Depends: R (>= 3.4.0)
 Imports: methods
 Suggests: bit64 (>= 4.0.0), bit (>= 4.0.4), R.utils, xts, zoo (>= 1.8-1), yaml, knitr, markdown
-Description: Fast aggregation of large data (e.g. 100GB in RAM), fast ordered joins, fast add/modify/delete of columns by group using no copies at all, list columns, friendly and fast character-separated-value read/write. Offers a natural and flexible syntax, for faster development.
+Description: Fast aggregation of large data (e.g. 100GiB in RAM), fast ordered joins, fast add/modify/delete of columns by group using no copies at all, list columns, friendly and fast character-separated-value read/write. Offers a natural and flexible syntax, for faster development.
```

Comment:
Oh, it feels too much, is it so much significant difference that we need to update description that was there for a decade(?) (or maybe even more?)

Reply:
To be fair, I believe for most readers GiB will be a bit confusing comparing to simply GB.

Reply:
funny use of "bit" :joy:

11 years: c2c8ed4936d89d1ac1d1656c6fa5cca565f15f7d

You're right this doesn't really matter (both 100GB and 100GiB are "large data"), this change is more with the thinking to enforce using IEC units everywhere.

But, for now at least, we'll already need carve-outs for `fwrite(buffMB=)` and `tables(mb=)`, so one more small carve-out makes no difference. Reverting.